{"name": "test_login_with_valid_email_and_invalid_password", "status": "failed", "statusDetails": {"message": "AssertionError: assert False\n +  where False = <method-wrapper '__eq__' of str object at 0x1053c5590>('Warning: No match for E-Mail Address and/or Password.')\n +    where <method-wrapper '__eq__' of str object at 0x1053c5590> = 'Warning: Your account has exceeded allowed number of login attempts. Please try again in 1 hour.'.__eq__\n +      where 'Warning: Your account has exceeded allowed number of login attempts. Please try again in 1 hour.' = display_warning_message()\n +        where display_warning_message = <pages.LoginPage.LoginPage object at 0x105383e10>.display_warning_message", "trace": "self = <tests.test_Login.TestLogin object at 0x105425910>\n\n    def test_login_with_valid_email_and_invalid_password(self):\n        home_page = HomePage(self.driver)\n        login_page = home_page.navigate_to_login_page()\n        login_page.login_to_application(\"alex.smith@yopmail.com\", \"P@ssw0rd123\")\n        time.sleep(2)\n        expected_message = \"Warning: No match for E-Mail Address and/or Password.\"\n>       assert login_page.display_warning_message().__eq__(expected_message)\nE       AssertionError: assert False\nE        +  where False = <method-wrapper '__eq__' of str object at 0x1053c5590>('Warning: No match for E-Mail Address and/or Password.')\nE        +    where <method-wrapper '__eq__' of str object at 0x1053c5590> = 'Warning: Your account has exceeded allowed number of login attempts. Please try again in 1 hour.'.__eq__\nE        +      where 'Warning: Your account has exceeded allowed number of login attempts. Please try again in 1 hour.' = display_warning_message()\nE        +        where display_warning_message = <pages.LoginPage.LoginPage object at 0x105383e10>.display_warning_message\n\ntests/test_Login.py:35: AssertionError"}, "start": 1749191166253, "stop": 1749191168928, "uuid": "cf3a98f3-1512-46d6-82ba-7c8c1ab9e4b7", "historyId": "0f7f188bae322a777580df5f4c12caa1", "testCaseId": "0f7f188bae322a777580df5f4c12caa1", "fullName": "tests.test_Login.TestLogin#test_login_with_valid_email_and_invalid_password", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup_and_teardown')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_Login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "Macmini"}, {"name": "thread", "value": "7441-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_Login"}]}